src/main.rs

```diff
@@ -95,18 +95,19 @@ fn main() {
 
     let (driver, _stop) = ZmqDriver::new();
 
-    driver.start(&args.endpoint, pbft_engine).unwrap_or_else(|err| {
-        error!("{}", err);
-        process::exit(1);
-    });
+    driver
+        .start(&args.endpoint, pbft_engine)
+        .unwrap_or_else(|err| {
+            error!("{}", err);
+            process::exit(1);
+        });
 }
 
 fn get_console_config(log_level: log::LevelFilter) -> Config {
     let stdout = ConsoleAppender::builder()
         .encoder(Box::new(PatternEncoder::new(
             "{h({l:5.5})} | {({M}:{L}):20.20} | {m}{n}",
-        )))
-        .build();
+        ))).build();
```

Comment:
I think these formatting fixes can get merged into the previous commit, it looks like that's the one that introduced this line

Reply:
Fixed